src/main/java/org/opensearch/security/dlic/rest/api/AuditApiAction.java

```diff
@@ -168,9 +168,13 @@ public AuditApiAction(final Settings settings,
         }
     }
 
-    @Override
     public List<Route> routes() {
```

Comment:
Why keep this function when you have deleted the same in other class files?

Reply:
My bad, they should be deleted as well, fixed